src/plugins/persistence/couch/scripts/lockObjects.mjs

```diff
@@ -0,0 +1,187 @@
+import debounce from 'p-debounce';
```

Comment:
Is there a reason you're not just using the couchdb node package to do this sort of traversal?

Reply:
No good reason, Couch CRUD operations are so simple it didn't occur to me to use a library. Glad you mentioned it though, nano actually has some useful features that have allowed me to simplify the code a lot, thanks.